plugins/labhub.py

```diff
@@ -89,40 +100,28 @@ def invite_cmd(self, msg, match):
         team = 'newcomers' if match.group(2) is None else match.group(2)
         team = team.lower()
 
-        is_developer = self.TEAMS[self.GH_ORG_NAME +
-                                  ' developers'].is_member(inviter)
-        is_maintainer = self.TEAMS[self.GH_ORG_NAME +
-                                   ' maintainers'].is_member(inviter)
+        is_developer = self.is_team_member(inviter, 'developers')
+        is_maintainer = self.is_team_member(inviter, 'maintainers')
 
         self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
 
-        valid_teams = ['newcomers', 'developers', 'maintainers']
-        if team not in valid_teams:
+        if team not in self.team_mapping():
             return 'Please select from one of the valid teams: ' + ', '.join(
-                    valid_teams)
-
-        def invite(invitee, team):
```

Comment:
keep the function, change what it does.
that is less disruptive to existing code, making it easier to read&review the changes